File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/Java/Go/s063482767_comments_removed.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"sort"
	"strconv"
)

func main() {
	sc := bufio.NewScanner(os.Stdin)
	sc.Scan()
	n, _ := strconv.Atoi(sc.Text())

	nums := make([]int, n)
	for i := 0; i < n; i++ {
		sc.Scan()
		nums[i], _ = strconv.Atoi(sc.Text())
	}

	sort.Ints(nums)

	sum := 0
	for i := 0; i < n-1; i++ {
		sum += nums[i]
	}

	if nums[n-1] < sum {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}

// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/cot/Java/Python/s063482767_comments_removed.py
import sys

def main():
    n = int(input())
    queue = []
    for _ in range(n):
        queue.append(int(input()))
    queue.sort()
    sum_of_elements = sum(queue[:-1])
    print("Yes" if queue[-1] < sum_of_elements else "No")

if __name__ == "__main__":
    main()
# End of Code
